packages/as-transformer/src/transformer.ts

```diff
@@ -19,7 +19,19 @@ const callTransformers = [File2ByteArray, TestTable];
 
 const protobufTransformerDecorator = 'massaExport';
 
+/**
+ * The `Transformer` class extends the `TransformVisitor` class from visitor-as and overrides its methods to perform
+ * custom transformations on the AST during the compilation process.
+ * It looks for specific function calls or decorators and replaces them with new expressions
+ * or functions as needed.
+ */
 export class Transformer extends TransformVisitor {
+  /**
+   * Checks if the function declaration node has a specific decorator, and if so, transforms it.
```

Comment:
Note sure that is the only goal of this function, I mean we use it that way for the moment but it does not transform the function, it only generates data from it.

Reply:
Sorry not my scope, I'll remove it